.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,109 @@
+name: Bug Report
+description: Report an issue or unexpected behavior that occurs within the application
+title: "[Bug]: "
+labels: "bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **README: Before You Submit Your Issue**
+        - Issues are not a place to go ask support questions or start discussions. Please ask support questions or start discussions on the [discussions page](https://github.com/FreeTubeApp/FreeTube/discussions).
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
+          required: true  
+        - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
+          required: true
+  - type: dropdown
+    attributes:
+      label: 'Issue Labels'
+      description: Please select a label that fits this bug report. Choose multiple, if applicable.
+      multiple: true
+      options:
+        - visual bug
+        - keyboard control not working
+        - text/string issue
+        - content not loading
+        - accessibility issue
+    validations:
+      required: true
```

Comment:
I don't think this drop down should be required

Reply:
im just following instructions :'(
https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-926196910